---
abstract: High-throughput sequencing of 16S rRNA amplicon has been extensively employed to perform microbiome characterization worldwide. As a culture-independent methodology, it has allowed high-level profiling of sample bacterial composition directly from samples. However, most studies are limited to information regarding relative bacterial abundances (sample proportions), ignoring scenarios in which sample microbe biomass can vary widely. Here, we use an equivolumetric protocol for 16S rRNA amplicon library preparation capable of generating Illumina sequencing data responsive to input DNA, recovering proportionality between observed read counts and absolute bacterial abundances within each sample. Under specified conditions, we show that the estimation of colony-forming units (CFU), the most common unit of bacterial abundance in classical microbiology, is challenged mostly by resolution and taxon-to-taxon variation. We propose Bayesian cumulative probability models to address such issues. Our results indicate that predictive errors vary consistently below one order of magnitude for total microbial load and abundance of observed bacteria. We also demonstrate our approach has the potential to generalize to previously unseen bacteria, but predictive performance is hampered by specific taxa of uncommon profile. Finally, it remains clear that high-throughput sequencing data are not inherently restricted to sample proportions only, and such technologies bear the potential to meet the working scales of traditional microbiology.
authors:
- admin
- Ana Paula Christoff
- Luiz Felipe Valter de Oliveira
author_notes:
- Equal contribution
- Equal contribution
date: "2021-02-01T00:00:00Z"
doi: "10.3389/fmicb.2021.638231"
featured: true
image:
  focal_point: ""
  preview_only: false
links:
- name: URL
  url: https://www.frontiersin.org/articles/10.3389/fmicb.2021.638231/full

publication: _Frontiers in Microbiology_
publication_short: ""
publication_types:
- "2"
publishDate: "2020-02-04T00:00:00Z"
summary: Library sizes obtained by 16S gene sequencing data are usually seen as arbitrary. But are they? Can we make them meaningful, both biologically and mathematically? Yes, we can.
tags:
- NGS
- Microbiome
- Bayesian statistics
- Cumulative Probability Models
- Ordinal Logistic Regression
title: Equivolumetric Protocol Generates Library Sizes Proportional to Total Microbial Load in 16S Amplicon Sequencing
url_pdf: https://www.frontiersin.org/articles/10.3389/fmicb.2021.638231/pdf
---

