---
abstract: Several studies have shown the ubiquitous presence of bacteria in hospital surfaces, staff, and patients. Frequently, these bacteria are related to HAI (healthcare-associated infections) and carry antimicrobial resistance (AMR). These HAI-related bacteria contribute to a major public health issue by increasing patient morbidity and mortality during or after hospital stay. Bacterial high-throughput amplicon gene sequencing along with identification of AMR genes, as well as whole genome sequencing (WGS), are biotechnological tools that allow multiple-sample screening for a diversity of bacteria. In this paper, we used these methods to perform a one-year cross sectional profiling of bacteria and AMR genes in adult and neonatal intensive care units (ICU and NICU) in a Brazilian public, tertiary hospital. Our results showed high abundances of HAI-related bacteria such as S. epidermidis, S. aureus, K. pneumoniae, A. baumannii complex, E. coli, E. faecalis, and P. aeruginosa in patients and hospital surfaces. Most abundant AMR genes detected throughout ICU and NICU were mecA, blaCTX-M-1 group, blaSHV-like, and blaKPC-like. We found that NICU environment and patients were more widely contaminated with pathogenic bacteria than ICU. Patient samples, despite the higher bacterial load, have lower bacterial diversity than environmental samples in both units. Finally, we also identified contamination hotspots in the hospital environment showing constant frequencies of bacterial and AMR contamination throughout the year. Whole genome sequencing (WGS), 16S rRNA oligotypes, and AMR identification allowed a high-resolution characterization of the hospital microbiome profile.
authors:
- Ana Paula Christoff
- Aline Fernanda Rodrigues Sereia
- admin
- Daniela Carolina de Bastiani
- Vanessa Leitner Silva
- Camila Hernandes
- Ana Paula Metran Nascente
- Ana Andrea dos Reis
- Renata Gon√ßalves Viessi
- Andrea dos Santos Pereira Marques
- Bianca Silva Braga
- Telma Priscila Lovizio Raduan
- Marines Dalla Valle Martino
- Fernando Gatti de Menezes
- Luiz Felipe Valter de Oliveira

date: ""
doi: "https://doi.org/10.1371/journal.pone.0234127"
featured: false
image:
  focal_point: ""
  preview_only: false
links:
- name: URL
  url: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234127

publication: _Plos One_
publication_short: ""
publication_types:
- "2"
publishDate: "2020-06-03T00:00:00Z"
summary: Bacteria are ubiquitous in hospital environments, in many cases threatening infection control. In this study, we use next-generation sequencing to track down these bacteria in intensive care units (ICUs) from a Brazilian hospital for a one-year period, covering environmental samples, hospital staff, and patients. We keep track of neonatal and adult ICUs and report abundance and diversity results as well as microbial resistance profiles.
tags:
- NGS
- Microbiome
- Hospital Microbiome
- Antimicrobial Resistance
title: One year cross-sectional study in adult and neonatal intensive care units reveals the bacterial and antimicrobial resistance genes profiles in patients and hospital surfaces
url_pdf: https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0234127&type=printable
---

